packages/pds/src/sequencer/events.ts

```diff
@@ -26,7 +26,8 @@ export const formatSeqCommit = async (
   if (writes.length > 200 || commitData.newBlocks.byteSize > 1000000) {
     tooBig = true
     const justRoot = new BlockMap()
-    justRoot.add(commitData.newBlocks.get(commitData.cid))
+    const rootBlock = commitData.newBlocks.get(commitData.cid)
+    rootBlock && justRoot.set(commitData.cid, rootBlock)
```

Comment:
I know this syntax works, but I prefer the explicit `if` statement 

```suggestion
if(rootBlock) {
  justRoot.set(commitData.cid, rootBlock)
}
```